src/cloud/docker/docker-extend.md

```diff
@@ -46,4 +46,22 @@ If you want to update the container configuration and test iteratively, run the
 docker-compose up -d --force-recreate --build
 ```
 
+## Building the Magento Cloud Docker PHP Images
+When developing or extending the Magento Cloud Docker you may need to modify the PHP Images. Extensions and configuration can generally be handled using the container configuration. This method gives additional flexibility, and documents how these images are generated and then uploaded to Docker Hub.
```

Comment:
This method?  Method = rebuilding the PHP container ...    This method documents how these images....  Do you mean that the documentation provides instructions to customize an image, rebuild it, and upload it to Docker Hub?

Reply:
This method is how the images are built prior to uploading to dockerhub. Using this method allows users to modify and rebuild and then they can use the images locally, this is described in another spot.

This is not documented anywhere yet, so if someone wanted to modify these, or even make changes that they are pushing back to Magento via a PR they will need this documentation to know how to do this correctly.